source/sass/wagtail/blocks.scss

```diff
@@ -37,19 +37,18 @@
   .streamfield-content {
     @extend .col-lg-8;
     &.wide {
-      @extend .col-12;
+      @apply tw-relative tw-w-full tw-px-8 tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full tw-max-w-full;
     }
     &.full-width {
-      @extend .col-12;
+      @apply tw-relative tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full;
```

Comment:
This one is a special case. 

I converted all CSS properties Boostrap's `.col-12` has into Tailwind classes and removed the ones that would override our custom `.full-width` rules.